modules/ossm-rn-new-features.adoc

```diff
@@ -19,11 +19,82 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 2.2
+
+This release of {SMProductName} adds new features and enhancements, and is support on OpenShift Container Platform 4.6 EUS, 4.7, 4.8, 4.9, and 4.10.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
+
+|===
+|Component |Version
+
+|Istio
+|1.12
```

Comment:
Should add "z" number for 1.12....e.g. Is is it 1.12.8?

Reply:
@longmuir looks like the release readiness doc was updated and shows. 1.12.7, so I've updated to that.